README.md

```diff
@@ -1,3 +1,317 @@
+* `validate`
+	* Function that takes in the value and returns `True` or `False` appropriately.
+* `transform`
+	* Function that takes in the value and returns a value of the same type. This will run _before_ validation.
+* `many`
+	* Defaults to `False`. Pass in `True` if multiple values are allowed for the field. `many=True` will result in a list of values on the parsed params/payload.
+	* For query parameters, this will support both comma separated values under a single arg (`?order_ids=1,2`) and multiple instances of the arg `(?order_ids=1&order_ids=2)`.
+	* For payloads, a list of values is expected.
+	* When used with `validate` and/or `transform`, these functions will be applied to each individual value.
+* `param_name`
+	* Use sparingly. It's to be used if the incoming query parameter will have a different name than the field name in the schema. For example, if the incoming query parameter will be specified as `?state=foo`, but you would like it to be `order_state` under `request.params`, you would do: `{"order_state": fields.String(param_name="state")}`
+
+The available fields and their custom arguments are:
+* `Bool`
+* `DateTime`
+* `Date`
+* `Time`
+* `Int`
+	* `min_val`
+	* `max_val`
+* `String`
+	* `min_length`
+	* `max_length`
+	* `allow_empty`
+		* Defaults to `False`. If you would like to accept empty strings as valid values passed in, do `allow_empty=True`.
+* `Enum`
+	* `choices`
+		* List of acceptable string values
+* `UUID`
+* `Float`
+* `Decimal`
+	* `max_digits`
+* `Email`
+* `Dict`
+	* Use sparingly. Allows for any arbitrary dictionary to be passed in.
+
+### Nested schemas
+
+To nest a schema, do:
+```python
+@view(payload={
+    "id": fields.Int(),
+    "nested_object": fields.Nested({
+        "name": fields.String()
+    })
+})
+```
+`Nested` fields accept all of the same arguments as the other fields.
+
+### What about PATCH?
+
+If you would like to support traditional PATCH behavior (where the client only sends down the fields to be updated), and need to know which fields were sent down or not, then doing something like `default=None` is not going to work for you. Normally, a field is either required, or has a default value and will appear on the parsed data with the default value if it was not sent down. Luckily, you can pass `default=fields.OMIT` to the field and it will be left out of the parsed data if it wasn't sent down. Here's an example:
+```python
+@view(
+    "PATCH",
+    payload={
+       "first_name": fields.String(default=fields.OMIT),
+        "last_name": fields.String(default=fields.OMIT)
+    }
+)
+def update_person(request, id=None):
+    person = Person.objects.get(id=id)
+
+    data = request.payload.as_dict()
+
+    if "first_name" in data:
+        person.first_name = data["first_name"]
+    if "last_name" in data:
+        person.last_name = data["last_name"]
+
+    person.save()
+
+    return responses.DetailResponse()
+
+```
+
+## Pagination
```

Comment:
Can a client request a page_size through a query param?

Reply:
Yup! Lemme call that out explicitly

Reply:
🙌